tests/TestRvsdgs.hpp

```diff
@@ -9,9 +9,9 @@
 #include <jlm/ir/operators.hpp>
 
 /**
-* \brief AliasAnalysisTest class
+* \brief RvsdgTest class
```

Comment:
The file is called TestRvsdgs, but the class is RvsdgTest.
I guess there's a logic that the file is intended as the collection of tests used to test Rvsdgs while RvsdgTest is one specific test.